container-images/intel-gpu/Containerfile

```diff
@@ -22,10 +24,10 @@ COPY --from=builder /llama-cpp/lib64/ /usr/local/lib64/
 COPY --from=builder /llama-cpp/include/ /usr/local/include/
 COPY intel-gpu/oneAPI.repo /etc/yum.repos.d/
 
-RUN dnf install -y intel-opencl libcurl lspci clinfo intel-oneapi-runtime-compilers intel-oneapi-mkl-core intel-oneapi-mkl-sycl-blas intel-oneapi-runtime-dnnl ; \
-    chown 0:0 /etc/passwd ; \
-    chown 0:0 /etc/group ; \
-    chmod g=u /etc/passwd /etc/group ; \
+RUN dnf install -y intel-opencl libcurl lspci clinfo intel-oneapi-runtime-compilers intel-oneapi-mkl-core intel-oneapi-mkl-sycl-blas intel-oneapi-runtime-dnnl && \
```

Comment:
I would suggest using additional "base" stage that install these libraries, and which is used both by build, and final container stages.

That way packages common to final and build stages would be installed only once.